src/Srclib/Types.hs

```diff
@@ -42,6 +47,20 @@ instance ToText LicenseScanType where
 instance ToJSON LicenseScanType where
   toJSON = toJSON . toText
 
+newtype OriginPath = OriginPath FilePath
```

Comment:
[nit] Please document the intention of this type.

Reply:
I've added a comment explaining what an `OriginPath` is and why they are that way.